toolchain/sem_ir/formatter.cpp

```diff
@@ -924,6 +918,31 @@ class Formatter {
     out_ << "\n";
   }
 
+  auto FormatPendingConstantValue(bool space_before) -> void {
```

Comment:
I was trying to figure out at call sites, is this saying there _is_ a space before the call, or that a space should be inserted _before_ the call?

I think as written, it's being told whether to add a space:

```suggestion
  auto FormatPendingConstantValue(bool needs_space_before) -> void {
```

But consider flipping the bool so that it'd be "has_space_before", since "has" is a typical form.

Reply:
Changed to an enum, hopefully this clarifies.